Options.py

```diff
@@ -688,6 +688,12 @@ class Range(NumericOption):
     range_start = 0
     range_end = 1
 
+    _RANDOM_OPTS = [
+        "random", "random-low", "random-middle", "random-high", 
+        "random-range-low-<min>-<max>", "random-range-middle-<min>-<max>"
```

Comment:
```suggestion
        "random-range-low-<min>-<max>", "random-range-middle-<min>-<max>",
```
Missing comma here causing it to concatenate with the next option. (Apparently I'm just destined to find this particular mistake.)

Reply:
I sure appreciate it though :)